src/admin-guide/index.rst

```diff
@@ -0,0 +1,8 @@
+Administration Guide
+====================
+
+
+.. toctree::
+   :maxdepth: 2
+
+   environments/example_envs.rst
```

Comment:
Can we put this in the reference section to avoid creating a new top-level section.

Reply:
If you like, but I have the admin-guide in #289 too.